src/Core/AppKernel.php

```diff
@@ -0,0 +1,416 @@
+<?php
+
+namespace SilverStripe\Core;
+
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use SilverStripe\Config\Collections\CachedConfigCollection;
+use SilverStripe\Control\Director;
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Control\HTTPResponse_Exception;
+use SilverStripe\Core\Cache\ManifestCacheFactory;
+use SilverStripe\Core\Config\ConfigLoader;
+use SilverStripe\Core\Config\CoreConfigFactory;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Injector\InjectorLoader;
+use SilverStripe\Core\Injector\SilverStripeServiceConfigurationLocator;
+use SilverStripe\Core\Manifest\ClassLoader;
+use SilverStripe\Core\Manifest\ClassManifest;
+use SilverStripe\Core\Manifest\ModuleLoader;
+use SilverStripe\Core\Manifest\ModuleManifest;
+use SilverStripe\Dev\DebugView;
+use SilverStripe\Dev\Install\DatabaseAdapterRegistry;
+use SilverStripe\Logging\ErrorHandler;
+use SilverStripe\ORM\DB;
+use SilverStripe\View\ThemeManifest;
+use SilverStripe\View\ThemeResourceLoader;
+
+class AppKernel extends CoreKernel
```

Comment:
I'd be happy to merge the classes, given CoreKernel by itself is useless. @sminnee want to weigh in?